contracts/README.md

```diff
@@ -6,4 +6,67 @@ DC contracts have compatible NameRecord structures from [D1DC (.1.country)](http
 2. Owners of ENS domains can pay a small fee to "reinstate" their records in DC contract. This means if someone purchases an ENS domain through a marketplace (such as an NFT marketplace), they can reclaim corresponding names and records in DC contract, i.e. to take over from previous owner of the ENS domain
 3. The DC contract's companion frontend (in `/client` folder) is meant to be used together so the user can rent a web3 and web2 domain simutaneously. 
 
-In other words, the DC contract is meant to provide some "extra service" on top of ENS contracts. Therefore, it charges some additional fees on top of purchasing an ENS domain from the ENS app. On a product level, the main value is to host and configure data that is also viewable in web2. The initial product only allows the renter to embed a tweet on their new domain, but more functionalities are coming soon
\ No newline at end of file
+In other words, the DC contract is meant to provide some "extra service" on top of ENS contracts. Therefore, it charges some additional fees on top of purchasing an ENS domain from the ENS app. On a product level, the main value is to host and configure data that is also viewable in web2. The initial product only allows the renter to embed a tweet on their new domain, but more functionalities are coming soon
+
+
```

Comment:
README is not TODO. Move dev notes to dev folder

Reply:
Moved to dev/metadata.md